src/fluree/db/api/query.cljc

```diff
@@ -165,15 +165,16 @@
       (let [assert-flakes  (not-empty (filter flake/op t-flakes))
             retract-flakes (not-empty (filter (complement flake/op) t-flakes))
 
-            asserts  (<? (json-ld-resp/flakes->res db cache compact fuel 1000000
-                                                   {:wildcard? true, :depth 0} 0 assert-flakes))
-            retracts (<? (json-ld-resp/flakes->res db cache compact fuel 1000000
-                                                   {:wildcard? true, :depth 0}
-                                                   0 retract-flakes))
+            asserts  (json-ld-resp/flakes->res db cache compact fuel 1000000
+                                               {:wildcard? true, :depth 0}
+                                               0 assert-flakes)
+            retracts (json-ld-resp/flakes->res db cache compact fuel 1000000
+                                               {:wildcard? true, :depth 0}
+                                               0 retract-flakes)
             ;; t is always positive for users
             result (cond-> {:t (- (flake/t (first t-flakes)))}
-                     asserts  (assoc :assert asserts)
-                     retracts (assoc :retract retracts))]
+                     asserts  (assoc :assert (<? asserts))
+                     retracts (assoc :retract (<? retracts)))]
```

Comment:
since there's always going to be an `asserts` and `retracts` channel this `cond->` will always include the associated keys, even if the result is empty.

Reply:
Yep, just noticed that too. Fix coming shortly